### ISSUE TITLE
Default emacs theme (again)

### ISSUE THREAD

**Issue by AlexDaniel** [hlissner/doom-emacs#5535]
### What did you expect to happen?

I want to use the default emacs color theme. Previously (after https://github.com/hlissner/doom-emacs/issues/4214 was resolved) I was able to do that by doing this:

```
(setq doom-theme 'default)
```

### What actually happened?

I'm getting this error:
```
*ERROR*: Unable to find theme file for ‘default’
```

### Steps to reproduce

1. Set your theme to default like this: `(setq doom-theme 'default)`
2. Run `doom sync`
3. Try starting emacs. It will fail with `*ERROR*: Unable to find theme file for ‘default’`

### System Information

https://gist.github.com/AlexDaniel/7e1e242ddcd6dcc27fe2da15bb695689

**Comment by AlexDaniel** [comment]
@hlissner that works, thanks!

**Comment by hlissner** [comment]
Glad I could help! Though, turns out `doom-theme`'s docstring was out of date, which I've fixed in fbfa144. Thanks for bringing it to my attention!
